lms/djangoapps/ora_staff_grader/ora_api.py

```diff
@@ -33,6 +34,36 @@ def get_submissions(request, usage_id):
     return json.loads(response.content)
 
 
+def get_assessments_given(request, usage_id, submission_uuid):
+    """
+    Get a list of assessments given from the ORA's 'list_assessments_given' XBlock.json_handler
+    """
+    handler_name = "list_assessments_given"
+    data = {"item_id": usage_id, "submission_uuid": submission_uuid}
+
+    response = call_xblock_json_handler(request, usage_id, handler_name, data)
+
+    if response.status_code != HTTPStatus.OK:
+        raise XBlockInternalError(context={"handler": handler_name})
```

Comment:
```suggestion
    if not HTTPStatus.OK:
        raise XBlockInternalError(context={"handler": handler_name})
```

Reply:
In this case, the `HttpResponse` object does not have an `ok` attribute, so it could only be evaluated using the `status_code`